src/app/src/pages/redteam/setup/components/Targets/TargetTypeSelection.tsx

```diff
@@ -152,35 +136,35 @@ export default function TargetTypeSelection({ onNext, onBack }: TargetTypeSelect
       }
     >
       <div className="flex flex-col gap-6">
-        {/* Quick Start Section */}
-        <Alert variant="info" className="[&>svg]:hidden">
-          <AlertDescription className="flex w-full flex-col gap-3">
-            <div className="flex items-center gap-3">
-              <Info className="h-4 w-4 shrink-0" />
-              <div className="flex min-w-0 flex-1 flex-wrap items-center justify-between gap-4">
-                <span className="min-w-[300px] flex-1 text-sm">
-                  <strong>New to Promptfoo</strong> and want to see it in action? Load an example
-                  configuration to get started immediately.
-                </span>
-                <LoadExampleButton />
-              </div>
-            </div>
-            <p className="text-sm text-muted-foreground">
-              <strong>Have an existing YAML config?</strong> Use the <strong>"Load Config"</strong>{' '}
-              button in the sidebar to import it and pre-fill the form.
-            </p>
-          </AlertDescription>
-        </Alert>
+        {/* Context + Quick Start */}
+        <div className="rounded-lg border border-border bg-muted/30 p-4">
+          <p className="text-sm text-foreground">
+            A target is the AI system you want to test—an agent, chatbot, API endpoint, or any{' '}
+            <a
+              href="https://www.promptfoo.dev/docs/providers/"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="text-primary hover:underline"
+            >
+              supported provider
+            </a>
+            .
+          </p>
+          <div className="mt-3 flex items-center gap-3">
+            <LoadExampleButton />
+            <span className="text-sm text-muted-foreground">New to Promptfoo? Try a demo</span>
+          </div>
+        </div>
```

Comment:
having this here instead of the header feels so weird, even from a text / element hierarchy perspective. Why is this a card?